columnphysics/icepack_parameters.F90

```diff
@@ -250,11 +250,12 @@ module icepack_parameters
          TTTocn = 5107.4_dbl_kind      ! for qsat over ocn
 
       character (len=char_len), public :: &
-         atmbndy = 'default' ! atmo boundary method, 'default' ('ccsm3') or 'constant'
+         atmbndy = 'default'           ! atmo boundary method, 'default' ('ccsm3') or 'constant'
 
       logical (kind=log_kind), public :: &
          calc_strair     = .true.  , & ! if true, calculate wind stress
          formdrag        = .false. , & ! if true, calculate form drag
+         heatflux_linear = .false. , & ! if true, calculate sensible+latent heatfluxes using traditional linear bulk formula
```

Comment:
As noted in the CICE PR, maybe there could be a third option for atmbndy instead of the new heatflux_linear flag.

Reply:
If a namelist/argument exists that we can leverage to set this new option, that's preferable to creating a new namelist/argument when it makes sense.

Reply:
Rewritten using 'atmbndy' namelist variable as suggested.
@eclare108213  I can't rename the PR removing 'traditional' but feel free to change it. Alternative I can create a new PR.